api/cmd/build/clean.go

```diff
@@ -56,6 +56,9 @@ func clean() {
 	r.FieldsPerRecord = 4
 
 	records, err := r.ReadAll()
+	if err != nil {
+		fmt.Printf("WARNING: %s\n", err)
```

Comment:
There should be a return after the warning so the code doesn't continue in an unknown state.